src/utils/localized.js

```diff
@@ -42,6 +42,14 @@ export const sortLanguages = (selectedLanguage, allLanguages) => {
   return [selectedLanguage, ...related.sort(), ...unrelated.sort()];
 };
 
+// sorts the currencies with the following priority:
+// - The selected currency is placed first (e.g EUR)
+// - All other currencies follow, sorted alphabetically as well
+export const sortCurrencies = (selectedCurrency, allCurrencies) => {
```

Comment:
It's kinda weird to have something about currencies in a file called `localized`. This file has so far been used for `LocalizedString` related stuff. I'd keep this functionality in the component itself.